hypervisor/arch/x86/pm.c

```diff
@@ -271,3 +273,81 @@ void reset_host(void)
 		asm_pause();
 	}
 }
+
+static enum acrn_cpufreq_policy_type cpufreq_policy = CPUFREQ_POLICY_PERFORMANCE;
+
+void init_frequency_policy(void)
+{
+	uint32_t cpuid_06_eax, unused;
+	struct acrn_boot_info *abi = get_acrn_boot_info();
+	const char *cmd_src = abi->cmdline;
+
+	/*
+	 * Parse cmdline, decide which policy type to use.
+	 * User can either specify cpu_perf_policy=Nominal or cpu_perf_policy=Performance
+	 * The default type is 'Performance'
+	 */
+	if(strstr_s(cmd_src, MAX_BOOTARGS_SIZE, "cpu_perf_policy=Nominal", 24U) != NULL) {
```

Comment:
Have we verified that this works correctly also when a command-line `uart=` parameter is present?

Reply:
I checked the code and did some extra test about this. And it works fine with 'uart=' parameter.